Sources/SwiftIDEUtils/FixItApplier.swift

```diff
@@ -36,6 +36,7 @@ public enum FixItApplier {
   ) -> String {
     let messages = messages ?? diagnostics.compactMap { $0.fixIts.first?.message.message }
 
+    // FIXME: This assumes every fix-it is applied to a node in the 'tree', which is not guaranteed.
```

Comment:
Could we check this and ignore Fix-Its that apply to different trees? Although I do worry that macro authors will have made Fix-Its that apply to modified trees and that we might break them with this…